frontend/src/pages/team/Devices/index.vue

```diff
@@ -37,6 +37,7 @@
                 </template>
             </div>
         </template>
+        <ff-loading v-else message="Loading Devices..." />
```

Comment:
This condition needs moving to the top so it takes precedence over the others. As it stands, it is never being reached.